components/configurator/ui/ConfiguratorPrompt.js

```diff
@@ -52,7 +43,7 @@ const ButtonIndicator = styled.div`
 `
 
 const ConfiguratorPrompt = ({ message, actionA, actionB }) => (
-  <Wrapper>
+  <ConfiguratorWrapper>
```

Comment:
I pulled this out into a separate component so I could use the same centering/layout for the configurator loading state.